### PR TITLE
Update version.c

### PR DESCRIPTION
**Pull request by Shougo**
1914: signal
2074: HIKEY2UF()

### CODE REVIEW

**Comment by jamessan** [comment]
2031 looks like something we should apply, possibly with some adjustments. We use runtest.vim.

**Comment by jamessan** [comment]
The offsetof patches also look useful.

**Comment by Shougo** [comment]
OK.  I get it.

> The offsetof patches also look useful.

**Comment by Shougo** [comment]
I have updated it.

File: src/nvim/version.c [hunk 1]
```diff
@@ -366,7 +366,7 @@ static int included_patches[] = {
   // 2077,
   // 2076,
   // 2075,
-  // 2074,
+  // 2074 NA
```
Comment (justinmk) [inline src/nvim/version.c]: this should be uncommented, not "NA"--we made an equivalent change.

**Comment by Shougo** [comment]
Thank you.

**Pull request closed without merging**